src/bitcoin-config.json

```diff
@@ -1,5 +1,7 @@
 {
-  "bitcoinPeerUri": "[FILL THIS IN!]",
+  "bitcoinPeerUri": "http://localhost:18332",
+  "bitcoinRpcUsername": "",
```

Comment:
Nit: Consider adding [FILL THIS IN!] for consistency.

Reply:
This was intentional. I had made RPC username and password optional, so leaving them blank will evaluate to false in javascript-type-conversion-anything-goes-land and thus RPC calls would be made without authorization headers.